CodeModel/src/main/java/org/openzen/zenscript/codemodel/member/EnumConstantMember.java

```diff
@@ -9,15 +9,17 @@
 	public final CodePosition position;
 	public final HighLevelDefinition definition;
 	public final String name;
+	public final String fieldName;
```

Comment:
Is that something that belongs in here, or would it somehow need to be mapped in the JAva-Shared/Bytecode module as this is Java-Specific?

Reply:
Where else could it be?

A lot of the code calls `.name`to get the field name and the constant name

Reply:
We could also have this mapped in a custom map or other structure inside the JavaBytecodeContext, since the field name is only actually required when compiling the calls.

But we could talk about this during our next ZC talk